src/pages/Admin/Venue/VenueAdminPage.tsx

```diff
@@ -48,13 +58,29 @@ export const VenueAdminPage: React.FC = () => {
     );
   }
 
-  const isIframeVenue = IFRAME_TEMPLATES.includes(venue.template);
-
   return (
-    <>
-      <h4 className="admin-page-title">You are editing venue: {venue.name}</h4>
-      <BannerAdmin venueId={venueId} venue={venue} />
-      {isIframeVenue && <IframeAdmin venueId={venueId} venue={venue} />}
-    </>
+    <WithNavigationBar>
```

Comment:
Why is `WithNavigationBar` being added here..?

Reply:
according to the design we should have the header bar & nav elements https://www.figma.com/file/oX3Px4vYjjUkOZYSqcCgWI/Sparkle-Banner-Announcements?node-id=3%3A6
I suppose that `withnavigationbar` handles exactly that

Reply:
On the non admin side of things this is added in a single location, not in individual pages.

Not sure if that differs on the admin side, o think it might. Is there no ‘centralised higher location’ this would make sense to exist?

Reply:
yep, as I can see with the current implementation `<WithNavigationbar>` is added to each page where it's necessary
![image](https://user-images.githubusercontent.com/56254806/124261542-9424f180-db39-11eb-9bc4-f1d0b28d0117.png)